src/pkg/fault/item.go

```diff
@@ -83,3 +83,71 @@ func OwnerErr(cause error, id, name string) *Item {
 		Cause: cause.Error(),
 	}
 }
+
+// ---------------------------------------------------------------------------
+// Skipped Items
+// ---------------------------------------------------------------------------
+
+type skipCause string
+
+const SkipMalware skipCause = "malware_detected"
+
```

Comment:
Should there be another cause for unknown or unresolvable?

Reply:
Nope.  We shouldn't be skipping unknown cases.  Those should just be errors.  Skips should only occur for well-known, well documented conditions.